### PR TITLE
Lint TensorFlow: Ragged Tensors

### PR DESCRIPTION
**Pull request by 8bitmp3**
Splitting PR #1736 into smaller digestible bits. Thanks! @lamberta

### CODE REVIEW

**Comment by 8bitmp3** [comment]
The examples use "we" and "I", which may be causing the CI test ❎ (first person -> second person) 😊 @lamberta 

```
        "sentences = tf.ragged.constant([\n",
        "    [\"Let's\", \"build\", \"some\", \"ragged\", \"tensors\", \"!\"],\n",
        "    [\"We\", \"can\", \"use\", \"tf.ragged.constant\", \".\"]])\n",
...
        "paragraphs = tf.ragged.constant([\n",
        "    [['I', 'have', 'a', 'cat'], ['His', 'name', 'is', 'Mat']],\n",
        "    [['Do', 'you', 'want', 'to', 'come', 'visit'], [\"I'm\", 'free', 'tomorrow']],\n",
        "])\n",
```

**Comment by 8bitmp3** [comment]
Gently pinging @lamberta @MarkDaoust Thank you!

**Comment by lamberta** [comment]
Looks like a merge conflict and failing lint?

**Comment by 8bitmp3** [comment]
Yup, it's the `Fail | google::second_person | Prefer second person instead of first person: https://developers.google.com/style/person` error. Likely reason - it's feature, not a bug 🙂:

```
        "sentences = tf.ragged.constant([\n",
        "    [\"Let's\", \"build\", \"some\", \"ragged\", \"tensors\", \"!\"],\n",
        "    [\"We\", \"can\", \"use\", \"tf.ragged.constant\", \".\"]])\n",
        "print(sentences)"
```

File: site/en/guide/ragged_tensor.ipynb [hunk 1]
```diff
@@ -1068,7 +1026,7 @@
       "source": [
         "#### Concrete functions\n",
         "\n",
-        "[Concrete functions](https://www.tensorflow.org/guide/concrete_function) encapsulate individual traced graphs that are built by `tf.function`.  Starting with TensorFlow 2.3 (and in `tf-nightly`), ragged tensors can be used transparently with concrete functions.\n"
+        "[Concrete functions](https://www.tensorflow.org/guide/function#obtaining_concrete_functions) encapsulate individual traced graphs that are built by `tf.function`. Ragged tensors can be used transparently with concrete functions."
```
Comment (8bitmp3) [inline site/en/guide/ragged_tensor.ipynb:1029]:
```suggestion
        "[Concrete functions](https://www.tensorflow.org/guide/function#obtaining_concrete_functions) encapsulate individual traced graphs that are built by `tf.function`. Ragged tensors can be used transparently with concrete functions.\n"
```

File: site/en/guide/ragged_tensor.ipynb [hunk 2]
```diff
@@ -1079,45 +1037,13 @@
...
-        "def decomposed_ragged_increment(x_values, x_splits):\n",
-        "  x = tf.RaggedTensor.from_row_splits(x_values, x_splits)\n",
+        "def increment(x):\n",
         "  return x + 1\n",
         "\n",
         "rt = tf.ragged.constant([[1, 2], [3], [4, 5, 6]])\n",
-        "cf = decomposed_ragged_increment.get_concrete_function(rt.values, rt.row_splits)\n",
-        "print(cf(rt.values, rt.row_splits))"
+        "cf = increment.get_concrete_function(rt)\n",
+        "print(cf(rt))"
```
Comment (8bitmp3) [inline site/en/guide/ragged_tensor.ipynb:1046]:
```suggestion
        "print(cf(rt))\n"
```

**Comment by 8bitmp3** [comment]
Hi @lamberta PTAL and please advise: if I address the nblint issue (use of first person), then the code has a merge conflict. If I address the merge conflict, then the code can't pass the CI/lint test 🙂

```
<<<<<<< lint-ragged-tensor
... If you need to use SavedModel signatures in a previous version of TensorFlow, then it's recommended that you decompose the ragged tensor into its component tensors."
=======
... If you need to use SavedModel signatures in a previous version of TensorFlow, then we recommend decomposing the ragged tensor into its component tensors."
>>>>>>> master
```

**Comment by lamberta** [comment]
Thanks, @8bitmp3 . I resolved the merge conflict to use your change

**Comment by 8bitmp3** [comment]
Hi @lamberta @MarkDaoust 👋  If this PR is good to go, can you please approve it and add the 'ready to pull' label? Thank you!

**Review by lamberta — APPROVED** [review]

**Comment by lamberta** [comment]
Thanks @8bitmp3 . Looks like it needs to be rebased with master

**Comment by 8bitmp3** [comment]
Done my best with rebasing with `master` @lamberta. I think `nblint` test is not passing because of the content in this cell/step where you construct a ragged tensor with `tf.ragged.constant`:

```py
sentences = tf.ragged.constant([
    ["Let's", "build", "some", "ragged", "tensors", "!"],
    ["We", "can", "use", "tf.ragged.constant", "."]])
print(sentences)
```

LMKWYT 👍

**Review by lamberta — APPROVED** [review]

**Comment by lamberta** [comment]
@8bitmp3 Still seeing a few merge conflicts, maybe something is off with your git workflow?

Make sure your local `master` branch is up-to-date and you've configured the repo like described here: https://www.tensorflow.org/community/contribute/docs#set_up_a_local_git_repo
Make changes on your feature branch and then rebase with the updated master branch.

**Pull request merged**